src/test/scala/mesosphere/marathon/api/v2/validation/AppValidationTest.scala

```diff
@@ -168,6 +169,32 @@ class AppValidationTest extends UnitTest with ValidationTestLike with TableDrive
       }
     }
 
+    "external volume" should {
```

Comment:
can we update AppValidationTest in a way that it checks both places in which validation was found to have occurred?